api/controllers/game/move.js

```diff
@@ -23,8 +23,8 @@ module.exports = async function (req, res) {
         throw new Error('You are not a player in this game!');
     }
 
-    validate(gameState, req.body, playedBy);
-    const updatedState = execute(gameState, req.body, playedBy);
+    validate(gameState, req.body, playedBy, game.gameStates);
+    const updatedState = execute(gameState, req.body, playedBy, game.gameStates);
```

Comment:
Sending prior (packed) `GameStateRow`s for the game as a 4th arg to both validate and execute to accommodate stalemate requests since:
1. Whether you can request a stalemate depends on whether one was requested previously this turn
2. When a stalemate request is rejected, we return to the phase from the previous move

Unfortunately this requires updating every `validate()` and `execute()` helper to accept this input, but I think we ultimately get the most consistent API as a result